tests/lisa/test_energy_model.py

```diff
@@ -0,0 +1,229 @@
+from collections import OrderedDict
+import unittest
+from unittest import TestCase
+
+
+from energy_model import (EnergyModel,
```

Comment:
Doesn't fit on a single line, 81 chars :D

The '(' is just the [standard way to do line continuation](https://www.python.org/dev/peps/pep-0008/#maximum-line-length).